app/embed/page.tsx

```
@@ -0,0 +1,138 @@
+  }, [state.post])
+
+  const viewPath = useMemo(() => {
+    if (!state.post) return '/blog'
+    return `/blog?blog=${encodeURIComponent(state.post.blogId)}&post=${encodeURIComponent(state.post.slug)}`
+  }, [state.post])
+
+  const createdLabel = state.post?.createdAt.toLocaleDateString() || ''
+
+  return (
+    <>
+      <style dangerouslySetInnerHTML={{ __html: EMBED_STYLES }} />
```

Biome (2.4.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

(lint/security/noDangerouslySetInnerHtml)